assets/js/app.js

```diff
@@ -10,14 +10,9 @@
 // Any SCSS/CSS you require will output into a single css file (app.css in this case)
 import '../scss/app.scss';
 
-import moment from 'moment';
+// eslint-disable-next-line import/no-unresolved, import/no-webpack-loader-syntax
```

Comment:
Should we remove some rule from eslint config ?

Reply:
I will take a look but probably is a problem with how the stimulus bridge works to lazy load a controller

Reply:
Disabled the no-webpack-loader-syntax, but I should no disable import/no-unresolved, since it will hide real issues. This is indeed a webpack loader syntax, perfectly fine since we are using webpack.